.github/workflows/ci.yml

```
@@ -28,6 +28,11 @@ on:
         type: boolean
         default: false
         required: true
+      run_bundle_tests:
+        description: 'Force run package tests'
```

Maybe be more explicit here?

```suggestion
        description: 'Force run package tests for releases'
```